Gemfile

```diff
@@ -1,7 +1,7 @@
 source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-ruby '2.7.0'
+ruby '>=2.6.6'
```

Comment:
- [ ] **Inconsisty ruby version:** your `Gemfile` and your `.ruby-version` have 2 different ruby version which course issues when running blunder, especially if the user has multiple environments